neural_modelling/src/neuron/plasticity/stdp/timing_dependence/timing_vogels_2011_impl.c

```diff
@@ -28,18 +28,25 @@ int16_lut *tau_lookup;
 //! Global plasticity parameter data
 plasticity_trace_region_data_t plasticity_trace_region_data;
 
+//! How the configuration data for vogels_2011 is laid out in SDRAM.
+typedef struct {
+    int32_t alpha;
+    uint32_t lut_data[];
+} vogels_2011_config_t;
+
 //---------------------------------------
 // Functions
 //---------------------------------------
 address_t timing_initialise(address_t address) {
     log_info("timing_initialise: starting");
     log_info("\tVogels 2011 timing rule");
+    vogels_2011_config_t *config = (vogels_2011_config_t *) address;
 
     // Copy parameters
-    plasticity_trace_region_data.alpha = (int32_t) address[0];
+    plasticity_trace_region_data.alpha = config->alpha;
 
     // Copy LUTs from following memory
-    address_t lut_address = &address[1];
+    address_t lut_address = config->lut_data;
```

Comment:
This is good; I wonder if it highlights a mistake below though!  I think this should return the address after reading the LUT.  Not changed here though so implies a mistake in master...

Reply:
The update to that is done by `maths_copy_int16_lut(&lut_address)` just below.

Reply:
Ah yes, phew!  I missed that it was passed by reference...